pkg/aws/services/iam/oidc.go

```diff
@@ -182,7 +182,6 @@ func (s *Service) EnsureOIDCProviders(identityProviderURLs []string, clientID st
 func (s *Service) internalTags() map[string]string {
 	return map[string]string{
 		key.S3TagOrganization: util.RemoveOrg(s.scope.ClusterNamespace()),
-		key.S3TagCluster:      s.scope.ClusterName(),
```

Comment:
I think the problem is that this is used by both vintage and CAPI and in vintage it does not have the `giantswarm.io/cluster` tag by default :(

Reply:
ok, I tried a different approach now.